Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -90,17 +89,99 @@ if(WIN32)
         win32/res/Dirt.ICO
         win32/res/headspin.ico
     )
+elseif(APPLE)
+    set(plClient_SOURCES ${plClient_SOURCES}
+        Mac-Cocoa/main.mm
+        Mac-Cocoa/PLSKeyboardEventMonitor.mm
+        Mac-Cocoa/PLSView.mm
+        Mac-Cocoa/PLSLoginWindowController.mm
+        Mac-Cocoa/PLSPatcherWindowController.mm
+        Mac-Cocoa/PLSPatcher.mm
+        Mac-Cocoa/PLSServerStatus.mm
+        Mac-Cocoa/StringTheory_NSString.mm
+    )
+    set(plClient_HEADERS ${plClient_HEADERS}
+        Mac-Cocoa/PLSKeyboardEventMonitor.h
+        Mac-Cocoa/PLSView.h
+        Mac-Cocoa/PLSLoginWindowController.h
+        Mac-Cocoa/PLSPatcherWindowController.h
+        Mac-Cocoa/PLSPatcher.h
+        Mac-Cocoa/PLSServerStatus.h
+        Mac-Cocoa/StringTheory_NSString.h
+    )
+    set(RESOURCES 
+        Mac-Cocoa/MainMenu.xib
+        Mac-Cocoa/PLSLoginWindowController.xib
+        Mac-Cocoa/PLSPatcherWindowController.xib
+        Mac-Cocoa/banner.png
+        Mac-Cocoa/banner@2x.png
+        win32/res/Dirt.ICO
+    )
+    
+    list(APPEND plClient_RESOURCES
+        win32/res/resource.h
+        win32/res/banner.bmp
+        win32/res/Dirt.ICO
+        win32/res/headspin.ico
+    )
 else()
     list(APPEND plClient_SOURCES
         main.cpp
     )
 endif()
 
-plasma_executable(plClient WIN32 CLIENT INSTALL_PDB
-    SOURCES
-        ${plClient_SOURCES} ${plClient_HEADERS}
-        ${plClient_TEXT} ${plClient_RESOURCES}
-)
+if(APPLE)
+    plasma_executable(plClient CLIENT INSTALL_PDB SOURCES ${plClient_SOURCES} ${plClient_HEADERS} ${RESOURCES})
+    set(MAIN_NIBFILE "MainMenu")
```

Comment:
What is `MAIN_NIBFILE` used for? It looks to be hardcoded in Info.plist

Reply:
This may be leftover from when CMake itself was generating the Info.plist. I'll remove it.